README.md

```diff
@@ -4,6 +4,8 @@
 [![Build Status](https://travis-ci.com/kelloggm/checkerframework-gradle-plugin.svg?branch=master)](https://travis-ci.com/kelloggm/checkerframework-gradle-plugin)
 
 This plugin configures `JavaCompile` tasks to use the [Checker Framework](https://checkerframework.org) for pluggable type-checking.
+The plugin runs checkers by creating a custom manifest file containing the checkers you specify,
```

Comment:
This implementation detail feels out of place on line 6 of the README.  I would cut it or leave it to later.